docs/monitoring-usage.rst

```diff
@@ -278,3 +278,112 @@ follows::
 
 .. _Time Series:
     https://cloud.google.com/monitoring/api/ref_v3/rest/v3/TimeSeries
+
+
+Writing Custom Time Series
+---------------------------
+
+The Stackdriver Monitoring API can be used to create and delete custom metrics, as well as write
+data points to them. Please refer to the documentation for the `Creating Custom Metrics`_
+documentation for more information.
+
+
+To create a custom TimeSeries value, you must first create a custom
+:class:`~gcloud.monitoring.metric.MetricDescriptor` object, and then use its
+type to create a fully parametrized :class:`~gcloud.monitoring.metric.Metric` by providing specific
+values for the available labels. You must create a fully parametrized  :class:`~gcloud.monitoring.resource.Resource` from one of the
+available ResourceDescriptors which can be listed using the :class:`~gcloud.monitoring.resource.ResourceDescriptor` class,
+which similarly requires that its available labels have values specified.
+
+>>> from gcloud import monitoring
+>>> import datetime
+>>> client = monitoring.Client()
+>>> label = LabelDescriptor('response_code', LabelValueType.INT64,
+    ...                         description='HTTP status code')
+>>> metric_descriptor = = client.metric_descriptor(
```

Comment:
There is an extra equal sign.

Also, the formatting is off in the lines below, it should just be a 4 space indent from L303.